### DISCUSSION TITLE
Promote certifications on the end of courses

### DISCUSSION THREAD

**Discussion by odysseuskir** [simplified-coding/simplified-coding#133] [Ιδέες / Σχόλια]
Simplified Coding offers certifications for Python and HTML/CSS, but they are not visible on the website. I propose we have a card on the end of each supported course, so people can get a certification.

**Comment by GalaxyGamingBoy** [comment]
hm why not like a lesson card. like we are doing with the lessons

**Comment by odysseuskir** [comment]
That could work

**Comment by Nikos1508** [comment]
We can do that until I make this "Revising lessons" in the end of each lesson. When I make them we can add them in these lessons (Revising). I think that this is better

**Reply by odysseuskir** [reply]
When did we discuss revision lessons? So the certifications will not be used?

**Reply by Nikos1508** [reply]
They will be used
But we will also have some revising courses. We can add a button about the certifications in these courses when I make them.

**Reply by odysseuskir** [reply]
You have not mentioned revision courses. Please create a new discussion.

**Reply by Nikos1508** [reply]
I did
